pkgs/by-name/kc/kcl/package.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "kcl";
```

```suggestion
  pname = "kcl-cli";
```

I recommend reserving the name kcl for other related packages that may be available in the future. This also makes the purpose of this package clearer.

---

fixed. PTAL